cmd/main.go

```diff
@@ -267,22 +267,21 @@ func waitSignal(cancelFuncs []context.CancelFunc) {
 }
 
 func useKMSAuth(c *config.Config) (*bind.TransactOpts, common.Address, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), c.EthTxManager.KMSConnectionTimeout.Duration)
 	defer cancel()
 
 	client, err := kms.NewKeyManagementClient(ctx)
 	if err != nil {
 		return nil, common.Address{}, fmt.Errorf("failed to create kms client: %w", err)
 	}
-	defer client.Close()
```

Comment:
Is this deliberately deleted? Its lifespan align with the `EthereumTransactor`?

Reply:
Yes that correct. Basically, the client needs to be used to sign transactions. Previously what was happening was the client would get created, cancelled and then returned so it would never work.